whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -112,7 +137,101 @@ <h3 class="title">Products</h3>
 </div>
 {% endblock content %}
 {% block footer %}
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
```

Comment:
And if we're going to use the 1.x series, try the latest in the 1.x series, 1.11.3:

https://developers.google.com/speed/libraries/#jquery